deblur/deblurring.py

```diff
@@ -55,7 +55,34 @@ def get_sequences(input_seqs):
     return seqs
 
 
-def deblur(input_seqs, read_error=0.05, mean_error=0.005, error_dist=None,
+def create_error_profile(read_error):
+    """Create a theoretical read error profile given
+    a maxima error rate
```

Comment:
Can you elaborate on what exactly the error profile is?  

For instance, how was the length of the error profile chosen?